src/Our.Umbraco.Ditto/ComponentModel/Processors/DittoProcessorRegistry.cs

```diff
@@ -31,6 +31,17 @@ internal class DittoProcessorRegistry
         private Type DefaultProcessorType = typeof(UmbracoPropertyAttribute);
 
         /// <summary>
+        /// The default post-processors.
+        /// </summary>
+        private List<DittoProcessorAttribute> DefaultPostProcessorAttributes = new List<DittoProcessorAttribute>()
```

Comment:
@mattbrailsford Do you think this should a `readonly` field? and what about making it `static`? given that the processor cache is static.

Reply:
Good point, yea, it should be and in fact, I think DefaultProcessorType should be static too (I know it's not a change related to this PR though so can be don't direct in develop branch, just mentioning it)

Reply:
I'll do the `DefaultProcessorType` one as a separate PR.  Can't be `readonly` though, as it can be replaced.